setup/product_origin/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
```

Comment:
Maybe encoding needed here as well as the other init files?

Reply:
In setup, this is not needed. This is generated by https://github.com/acsone/setuptools-odoo